docs/setup/embedding.asciidoc

```diff
@@ -0,0 +1,49 @@
+[[embedding]]
+== Embedding {kib}
+
+You can embed a {kib} dashboard or visualization using an HTML code snippet generated with the *Copy iFrame code* button in *Share > Embed code* menu in *Dashboard* or *Visualize*.
+
+NOTE: Embedding of any other part of {kib} is also generally possible, but you may need to craft the proper HTML code manually.
+
+[role="screenshot"]
+image::images/embed-kibana.png[Generate an HTML snippet to embed {kib}, align=center]
+
+[float]
+[[embedding-security]]
+=== Configure security
+
+Embedding through iframes, if used properly, does not directly pose a security risk, but still requires a careful consideration.
+
+[float]
+==== Authentication
+If you're embedding {kib} to the website that supports Single Sign-On with SAML, OpenID Connect, Kerberos or PKI it's highly advisable to configure {kib} as a part of this Single Sign-On setup as well. Operating within a single and properly configured security domain would provide you with the most secure and seamless user experience. You can read more at <<kibana-authentication, Authentication in {kib}>>.
+
+If you want anyone to access embedded {kib} skipping the login step, but the Single Sign-On isn't an option for you, consider configuring the <<anonymous-authentication, anonymous access>> instead. It is already natively integrated into the embedding workflow for the dashboards and visualizations.
+
+If you have multiple authentication providers enabled, and you want to automatically log in anonymous users when embedding anything other than dashboards and visualizations, then you will need to add the `auth_provider_hint=<anonymous-provider-name>` query string parameter to the {kib} URL that you're embedding.
```

Comment:
anything other > objects other than?

Reply:
I was thinking about things like monitoring application, or discover, or live logs - not necessarily a Saved Object, but rather any other part of the Kibana.

Reply:
sounds good as is.